core-bundle/src/Config/Loader/XliffFileLoader.php

```diff
@@ -42,14 +43,14 @@ public function load($file, $type = null): string
 
     public function supports($resource, $type = null): bool
     {
-        return 'xlf' === pathinfo((string) $resource, PATHINFO_EXTENSION);
+        return 'xlf' === Path::getExtension((string) $resource, true);
     }
 
     private function convertXlfToPhp(string $name, string $language): string
     {
         $xml = $this->getDomDocumentFromFile($name);
 
-        $return = "\n// ".str_replace(strtr($this->rootDir, '\\', '/').'/', '', strtr($name, '\\', '/'))."\n";
+        $return = "\n// ".str_replace(Path::normalize($this->rootDir).'/', '', Path::normalize($name))."\n";
```

Comment:
```suggestion
        $return = "\n// ".Path::makeRelative($name, $this->rootDir)."\n";
```
If I’m understanding the code correctly?

Reply:
Ah, yes of course.